src/components/OpportunityDetails/OpportunityDetails.tsx

```diff
@@ -0,0 +1,140 @@
+import { components } from '@/api/schema';
+import {
+  ActionButton,
+  FavoriteToggle,
+  LoginModal,
+  MainLayout,
+  RoutesNavigationList,
+  SimpleNavigationList,
+  Title,
+} from '@/components';
+import { ScrollHeading } from '@/components/ScrollHeading/ScrollHeading';
+import { useEnvironment } from '@/hooks/useEnvironment';
+import { type MahdollisuusTyyppi } from '@/routes/types';
+import { useToolStore } from '@/stores/useToolStore';
+import { getLocalizedText } from '@/utils';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { MdCompareArrows, MdOutlinePrint, MdOutlineRoute, MdOutlineShare } from 'react-icons/md';
+
+export interface OpportunityDetailsSection {
+  navTitle: string; // Navigation title
+  content: JSX.Element; // Section content
+  dev?: boolean; // Only render in dev environment
```

Comment:
These properties are quite self explanating, no might need comments.

And if these are wanted to have comments, let's use tje JSDoc style so they comments can be seen nicely in the IDE when using the interface.

E.g:
```
/** Navigation title */
navTitle: string;
```

Reply:
True, these are pretty self explanatory. I removed the comments and renamed `dev` -> `showInDevOnly` to make more clear